pom.xml

```diff
@@ -478,6 +478,12 @@
                 <groupId>com.googlecode.json-simple</groupId>
                 <artifactId>json-simple</artifactId>
                 <version>1.1.1</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>junit</groupId>
+                        <artifactId>junit</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
I have added junit as exclusion for now in scope of this ticket. If this solves the dependency problem, can I raise a separate ticket to remove json-simple?